Problem/.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
You can keep only one `.gitignore` file in the root of the project that will do the job.